cmds/core/tty/tty.go

```diff
@@ -29,7 +29,10 @@ func run(stdout io.Writer) error {
 	return filepath.WalkDir("/dev", func(path string, dir os.DirEntry, _ error) error {
 		switch path {
 		case "/dev/fd":
-			return filepath.SkipDir
+			// fd can be a symbolic link in this case whole /dev dir will be skipped
```

Comment:
I'm missing something, your test is for IsDir, but the comment is for symbolic link? Also, 
"in this case who /dev dir will be skipped"
is it /dev or /dev/fd?

Reply:
In old code I assume that /dev/fd is a directory, but in Linux (Ubuntu) it is `/dev/fd: symbolic link to /proc/self/fd` so tty command just skip the rest of /dev directory.

Reply:
ah, can I reword that comment?
// On some systems, /dev/fd is a directory. On Linux, /dev/fd is a symlink.
// If /dev/fd is a directory, it should not be walked, so return file path.SkipDir.

Reply:
sure, updated.